examples/docs_beta_snippets/docs_beta_snippets/dagster-plus/deployment/hybrid/agents/local_dagster.yaml

```
@@ -0,0 +1,11 @@
+instance_class:
+  module: dagster_cloud.instance
+  class: DagsterCloudAgentInstance
+
+dagster_cloud_api:
+  agent_token: <YOUR_AGENT_TOKEN>
```

Can we switch this to:
```suggestion
  agent_token:
      env: DAGSTER_AGENT_TOKEN
```

We should resist encouraging users to hardcode a token in their config file and should instead recommend they read from elsewhere (in this example, an environment variable named `$DAGSTER_AGENT_TOKEN`)

---

what other ways can `dagster.yaml` source the token?

---

It's just https://docs.dagster.io/_apidocs/config#dagster.StringSource so I guess hardcoded or env? But there are other ways to set the agent token in particular - for example:

```
dagster-cloud agent run --agent-token
```